packages/js-client/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "@aragon/sdk-client-template",
+  "author": "Aragon Association",
+  "version": "1.0.0",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "engines": {
+    "node": ">=16"
+  },
+  "scripts": {
+    "start": "tsdx watch",
+    "build": "tsdx build",
+    "test": "TESTING=true tsdx test --verbose --runInBand",
```

Comment:
I get some warnings

```
 PASS  test/integration/methods.test.ts
  Methods
    ✓ should test a method (3ms)

Test Suites: 5 passed, 5 total
Tests:       14 passed, 14 total
Snapshots:   0 total
Time:        9.731s
Ran all test suites.
  console.warn
    Method not implemented.

       8 |   // implementation of the methods in the interface
       9 |   public myMethod() {
    > 10 |     console.warn('Method not implemented.');
         |             ^
      11 |   }
      12 | }
      13 |

      at SimpleStorageClientMethods.Object.<anonymous>.SimpleStorageClientMethods.myMethod (src/internal/modules/methods.ts:10:13)
      at test/integration/methods.test.ts:19:20
      at step (node_modules/tslib/tslib.js:193:27)
      at Object.next (node_modules/tslib/tslib.js:174:57)
      at node_modules/tslib/tslib.js:167:75
      at Object.__awaiter (node_modules/tslib/tslib.js:163:16)

✨  Done in 10.81s.
```

Reply:
Yes, I left some warnings when calling the methods, since it's just a template I wanted to have some methods examples butt without doing anything

Reply:
Is this still happening now?

Reply:
nope